commit 9671640ba1b3f248c207c377376cfb304b6157e4 (tag: v2.2.1, refs/changes/55/855/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-11-08T13:44:58-08:00 (2 years, 11 months ago)
    
    joyent/node-fast#3 client should not log at "error" level
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 8a4ba05..f9c4f53 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,10 @@
 
 No changes.
 
+## v2.2.1
+
+* #3 client should not log at "error" level
+
 ## v2.2.0
 
 * #2 update dependencies to support Node v4
diff --git a/lib/fast_client.js b/lib/fast_client.js
index e719f8f..b21b12d 100644
--- a/lib/fast_client.js
+++ b/lib/fast_client.js
@@ -486,7 +486,7 @@ FastClient.prototype.allocMessageId = function ()
  */
 FastClient.prototype.fatalError = function (err)
 {
-	this.fc_log.error(err);
+	this.fc_log.warn(err);
 	this.fc_nerrors++;
 
 	if (this.fc_error !== null) {
diff --git a/package.json b/package.json
index b675ef8..afe3729 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "fast",
 	"description": "streaming JSON RPC over TCP",
-	"version": "2.2.0",
+	"version": "2.2.1",
 	"main": "./lib/fast.js",
 	"repository": {
 		"type": "git",
